src/templates/base.html.twig

```diff
@@ -0,0 +1,230 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>{% block title %}Welcome!{% endblock %}</title>
+        <link rel="icon" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 128 128%22><text y=%221.2em%22 font-size=%2296%22>⚫️</text><text y=%221.3em%22 x=%220.2em%22 font-size=%2276%22 fill=%22%23fff%22>sf</text></svg>">
+        {% block stylesheets %}
+        {% endblock %}
+
+        {% block javascripts %}
+            {% block importmap %}{{ importmap('app') }}{% endblock %}
+        {% endblock %}
+    </head>
+    <body>
+    {% block header  %}
+<div id="header">
+  <div class="container">
+    <nav>
+    {% block logo %}
+      <a href="#" class="logo">Josef.</a>
+      {% endblock logo %}
+      <ul id="sidemenu">
+     
+        <li><a href="#header">Domů</a></li>
+        <li><a href="#about">O mně</a></li>
+        <li><a href="#services">Služby</a></li>
+        <li><a href="#portofolio">Portofolio</a></li>
+        <li><a href="#contact">Kontakt</a></li>
+        <i class="fa-solid fa-xmark" onclick="closemenu()"></i>
+      </ul>
+      <i class="fa-solid fa-bars" onclick="openmenu()"></i>
+    </nav>
+     {% endblock header %}
+    <div class="header-text">
+    {% block body %}
+      <p>Web developer & PPC specialista</p>
+      <h1>Ahoj! Jmenuji se <span>Josef</span><br>a zaručuji nejlepší webové stránky!</h1>
+    </div>
+  </div>
+  {% endblock body %}
+</div>
```

Comment:
The indentation is slightly broken (it doesn't affect functionality, it's just hard to read and easy to make a mistake)